### ISSUE TITLE
[DOC] list of all PODXXXX variables

### ISSUE THREAD

**Issue by iulian0512** [CocoaPods/CocoaPods#11348]
Where can i find a list of all POD... variables like
PODS_CONFIGURATION_BUILD_DIR
POD_ROOT 
etc

**Comment by orta** [comment]
Good idea, from a quick look in the codebase there's not a single central place listing them all - it'd be a good community contribution to document these 👍🏻
